11ty/definitions/nl_NL/biseksueel.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Biseksueel
+slug: biseksueel
+speech: adj
+defined: true
+---
+
+het zijn, relateren aan of zich laten kenmerken door seksueel aangetrokken zijn tot meer dan een geslacht.
```

Comment:
Did you mean "één" here rather than "een"?
Also, "geslacht" translates to "sex" in English, but don't we mean gender here, so that would be "gender"?

Reply:
I am not totally sure it needs to be accented in this sentence? See also https://onzetaal.nl/taaladvies/een-van-beiden/

Reply:
The way I read it without the accent is as "a" gender, not "one" gender

Reply:
It indeed should be één instead of een. That distinction is important. Will edit it!